examples/js/loaders/GLTFLoader.js

```diff
@@ -3129,16 +3129,16 @@ THREE.GLTFLoader = ( function () {
 
 		var nodeDef = json.nodes[ nodeIndex ];
 
-		return new Promise( function ( resolve ) {
+		return function( ) {
```

Comment:
I think we'll need to reorganize this a little at some point — would be nice to flatten it and avoid the IIFE here, and also see https://github.com/mrdoob/three.js/issues/14875. But this looks like an improvement for now. 👍

Reply:
actually, for now could you just wrap the IIFE in parentheses, with no space between the `()`?

```js
( function() {
  // ...
}() )
```

I find that makes it easier to see we've returned a promise, not a function. Thanks!

Reply:
Done, thanks!